starboard/android/apk/app/src/main/java/dev/cobalt/account/AccessToken.java

```diff
@@ -1,43 +0,0 @@
-// Copyright 2017 The Cobalt Authors. All Rights Reserved.
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-package dev.cobalt.account;
-
-import dev.cobalt.util.UsedByNative;
-
-/** POJO holding the token for a signed-in user. */
-public class AccessToken {
```

Comment:
this file can not be deleted. we have reference of it here https://source.corp.google.com/piper///depot/google3/java/com/google/android/libraries/youtube/tv/services/accountmanager/AccountManager.java;l=260?q=dev.cobalt.account.AccessToken
we need to move it to google3 before it delete it.

Reply:
Ack, i'll back this out

Reply:
you do not have to.